client/my-sites/checkout/composite-checkout/components/refund-policies.tsx

```diff
@@ -222,6 +266,13 @@ function RefundPolicyItem( { refundPolicy }: { refundPolicy: RefundPolicy } ) {
 			);
 			break;
 
+		case RefundPolicy.GiftYearlyPurchase:
+			text = translate(
+				'You understand that gift {{refundsSupportPage}}refunds{{/refundsSupportPage}} are limited to 14 days after purchase for products with yearly subscriptions.',
```

Comment:
For this and the others, should we be saying "for non-domain products" (rather than just "for products") to match how it works for non-gifts?

Reply:
Thanks for catching this. I had originally removed `non-domain` because I had planned to group domains under annual products. Since I added a domain-specific refund message, it makes sense to add it back in. Done.